### PR TITLE
Update name

### PR DESCRIPTION
**Pull request by justinkaseman**

### CODE REVIEW

File: FunctionsRequestSimulator/Functions.js [hunk 1]
```diff
@@ -4,20 +4,20 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
 };
```
Comment (KuphJr) [inline FunctionsRequestSimulator/Functions.js:4]: This is auto-generated JS code from TypeScript.  It was already reviewed in the `update-name` PR for the sandbox, so it should not have to be reviewed line-by-line here.

File: FunctionsRequestSimulator/Sandbox.js [hunk 2]
```diff
@@ -9,22 +9,22 @@ const os_1 = __importDefault(require("os"));
 const path_1 = __importDefault(require("path"));
```
Comment (KuphJr) [inline FunctionsRequestSimulator/Sandbox.js:9]: This is auto-generated JS code from TypeScript. It was already reviewed in the update-name PR for the sandbox, so it should not have to be reviewed line-by-line here.

**Review by KuphJr — APPROVED** [review]

File: Functions-request-config.js [hunk 3]
```diff
@@ -48,8 +48,4 @@ const requestConfig = {
   expectedReturnType: ReturnType.uint256,
 }
 
-if (requestConfig.secrets && !requestConfig.walletPrivateKey) {
```
Comment (bolekk) [inline Functions-request-config.js:51]: Why is this removed? Shouldn't we always require private key to be set?

File: .gitignore [hunk 4]
```diff
@@ -102,4 +102,4 @@ gas-report.txt
 
 contracts/test/fuzzing/crytic-export
 
-on-demand-request.json
\ No newline at end of file
+Functions-request.json
```
Comment (bolekk) [inline .gitignore:105]: nit: Not a huge fan of uppercase filenames but not a big deal.

File: hardhat.config.js [hunk 5]
```diff
@@ -20,17 +20,19 @@ if (MAINNET_RPC_URL !== "https://eth-mainnet.alchemyapi.io/v2/your-api-key") set
...
 if (MUMBAI_RPC_URL !== "https://polygon-mumbai.g.alchemy.com/v2/v2/your-api-key") setRpcUrlCount++
-if (setRpcUrlCount > 1) {
-    throw Error(
-        'Only 1 of the following environment variables can be set: MAINNET_RPC_URL, GOERLI_RPC_URL, POLYGON_MAINNET_RPC_URL, or MUMBAI_RPC_URL'
-    )
-}
 if (setRpcUrlCount === 0) {
     throw Error(
-        '1 of the following environment variables must be set: MAINNET_RPC_URL, GOERLI_RPC_URL, POLYGON_MAINNET_RPC_URL, or MUMBAI_RPC_URL'
+        'One of the following environment variables must be set: MAINNET_RPC_URL, GOERLI_RPC_URL, POLYGON_MAINNET_RPC_URL, or MUMBAI_RPC_URL'
```
Comment (bolekk) [inline hardhat.config.js:25]: "At least one"

File: hardhat.config.js [hunk 6]
```diff
@@ -20,17 +20,19 @@ if (MAINNET_RPC_URL !== "https://eth-mainnet.alchemyapi.io/v2/your-api-key") set
 if (POLYGON_MAINNET_RPC_URL !== "https://polygon-mainnet.alchemyapi.io/v2/your-api-key") setRpcUrlCount++
```
Comment (bolekk) [inline hardhat.config.js:20]: What's the point of all those different default strings? Just use one constant "UNSET" or something like that.

File: hardhat.config.js [hunk 7]
```diff
@@ -20,17 +20,19 @@ if (MAINNET_RPC_URL !== "https://eth-mainnet.alchemyapi.io/v2/your-api-key") set
...
-}
 if (setRpcUrlCount === 0) {
     throw Error(
-        '1 of the following environment variables must be set: MAINNET_RPC_URL, GOERLI_RPC_URL, POLYGON_MAINNET_RPC_URL, or MUMBAI_RPC_URL'
+        'One of the following environment variables must be set: MAINNET_RPC_URL, GOERLI_RPC_URL, POLYGON_MAINNET_RPC_URL, or MUMBAI_RPC_URL'
     )
 }
 
+const getChainToFork = () => {
+    if (MAINNET_RPC_URL !== 'https://eth-mainnet.alchemyapi.io/v2/your-api-key') return MAINNET_RPC_URL
```
Comment (bolekk) [inline hardhat.config.js:30]: Use a single constant, not 4 different (and useless) strings.

File: package.json [hunk 8]
```diff
@@ -54,5 +52,12 @@
       "prettier --write",
       "solhint"
     ]
+  },
+  "prettier": {
+    "trailingComma": "es5",
+    "tabWidth": 2,
```
Comment (bolekk) [inline package.json:58]: Thanks!

**Review by bolekk — APPROVED** [review]
Looks good!

File: hardhat.config.js (hunk 6 above)
Comment (KuphJr) [reply hardhat.config.js:20]: Yeah they were a holdover from the original tooling repo, but I agree they don't make much sense.

File: .gitignore (hunk 4 above)
Comment (KuphJr) [reply .gitignore:105]: We can address later if necessary

File: Functions-request-config.js (hunk 3 above)
Comment (KuphJr) [reply Functions-request-config.js:51]: This check is already handled within `hardhat-config.js`
